src/controller/PostController.php

```diff
@@ -1,9 +1,9 @@
 <?php
```

Comment:
il manque cette ligne : 
```suggestion
declare(strict_types=1)
```

Reply:
https://www.babeuloula.fr/blog/a-quoi-sert-declare-strict-types-1-en-php.html

Reply:
A reproduire sur tous les fichiers PHP :D

Reply:
fait mais après le commentaire de fichier
A voir si c'est au bon endroit